src/main/java/org/openelisglobal/common/provider/query/TestResultLimitsProvider.java

```diff
@@ -107,6 +107,10 @@ public int compare(ResultLimit e1, ResultLimit e2) {
                 resultLimitObject.put("highNormal", resultLimit.getHighNormal());
                 resultLimitObject.put("lowValid", resultLimit.getLowValid());
                 resultLimitObject.put("highValid", resultLimit.getHighValid());
+                resultLimitObject.put("highCriticalRangeHigh", resultLimit.getHighCriticalRangeHigh());
```

Comment:
@gitcliff , i think we agrred with @caseyi to refactor this abit .
We don't really need to define new High and Low critical **Ranges** explicitly ie
`highCrticalRangeHigh-HighCrticalRangeLow` . and `lowCrticalRangeHigh-lowCrticalRangeLow`.
Technically we already define min and Max valid range ie `highValid-lowValid`

So in summary to define critical rages , you only need to define only High critical and low critical values. ie `highCritical `and `lowCritica`l

- That way the low critical range becomes `lowValid-lowCrtical `instead of  `lowCrticalRangeHigh-lowCrticalRangeLow`

- The High critical range become `highValid-highCritical` instead of `highCrticalRangeHigh-HighCrticalRangeLow`

Reply:
That way will only need to validate that the high and low critical values are with in the valid range